src/account/ZulipLoginView.js

```diff
@@ -63,13 +63,18 @@ const ZulipLoginButton = (props) => (
     </TouchableHighlight>
 );
 
+ZulipLoginButton.propTypes = {
+  spinning: React.PropTypes.bool.isRequired,
+  onPress: React.PropTypes.func.isRequired,
+};
+
 export default class ZulipLoginView extends Component {
   constructor(props) {
     super(props);
     this.state = {
       realm: 'http://localhost:9991',
-      email: 'cordelia@zulip.com',
-      password: 'testing123',
+      email: 'test@test.com',
+      password: 'password',
```

Comment:
Could you remove this default dev user/password and replace with an empty string? This will likely vary between people's dev setups.